.circleci/config.yml

```diff
@@ -290,39 +290,21 @@ jobs:
 
   # Isolated theme build within a vanilla Drupal installation.
   # This replicates (to a degree) what Drupal CI would do.
-  build-isolated-php-82:
-    <<: *runner_config
-    docker:
-      - image: cimg/php:8.2-browsers
-    environment:
-      DRUPAL_VERSION: 10.2
-      DRUPAL_PROJECT_SHA: 10.x
-    <<: *job-build-isolated
-
-  build-isolated-php-82-legacy:
```

Comment:
@richardgaunt Not sure/ aware of how to remove from running in the CI and also how to add 
build-isolated-php-84

Reply:
@joshua-salsadigital Lets talk tomorrow I am not sure either on first inspection